.github/actions/nm-build-vllm-whl/action.yml

```diff
@@ -24,11 +30,19 @@ runs:
       source $(pyenv root)/versions/${{ inputs.python }}/envs/${VENV}/bin/activate
       SUCCESS=0
       pip3 wheel --no-deps -w dist . || SUCCESS=$?
-      echo "status=${SUCCESS}" >> "$GITHUB_OUTPUT"
       ls -alh dist/
       BASE=$(./.github/scripts/convert-version ${{ inputs.python }})
       WHL_FILEPATH=$(find dist -iname "*nm_vllm*${BASE}*.whl")
       WHL=$(basename ${WHL_FILEPATH})
       echo "whl=${WHL}" >> "$GITHUB_OUTPUT"
+      # generate .tar.gz
+      $SUCCESS || python3 setup.py sdist || SUCCESS=$?
```

Comment:
Hey Dan. The `$SUCCESS ||` in `$SUCCESS || python3 setup.py sdist || SUCCESS=$?` isn't doing what is intended. I see 
```
/opt/actions-runner/_work/_temp/68fcf277-8944-46d0-8362-4c8020852b3b.sh: line 12: 0: command not found
```
in https://github.com/neuralmagic/nm-vllm/actions/runs/8359879522/job/22884186408
I think it is safe to remove.

Reply:
Hi Varun, the $SUCCESS in $SUCCESS || python3 setup.py sdist || SUCCESS=$? is to keep the value in SUCCESS if the previous wheel generation step failed. It'll only run the tarball generation only if $SUCCESS=0, i.e. the wheel generation step is successful.

Reply:
Updated command so it'll work as intended, good catch.

Reply:
Thanks Dan.